velox/functions/lib/Re2Functions.h

```diff
@@ -98,6 +98,14 @@ std::shared_ptr<exec::VectorFunction> makeRe2Extract(
 
 std::vector<std::shared_ptr<exec::FunctionSignature>> re2ExtractSignatures();
 
+// TODO: Write comments
```

Comment:
Did you miss this?

Reply:
Yes, new commits have comments. Thank you